openai.moderations/Module.md

```diff
@@ -0,0 +1,81 @@
+This is a generated connector for [OpenAI Moderations API](https://beta.openai.com/docs/api-reference/moderations) OpenAPI Specification. OpenAI is an American artificial intelligence research laboratory consisting of a non-profit corporation and a for-profit subsidiary. OpenAI conducts AI research with the declared intention of promoting and developing friendly AI. The OpenAI Moderations API provides a way to access new moderations models developed by OpenAI for content moderation tasks.
```

Comment:
```suggestion
This is a generated connector for the [OpenAI `moderations` API](https://beta.openai.com/docs/api-reference/moderations) OpenAPI Specification. OpenAI is an American artificial intelligence research laboratory consisting of a non-profit corporation and a for-profit subsidiary. OpenAI conducts AI research with the declared intention of promoting and developing friendly AI. The OpenAI Moderations API provides a way to access new moderation models developed by OpenAI for content moderation tasks.
```
For the name of the API, keep the first letter in lowercase and have the name within backticks. Please change in all occurrences.